lesson-2/idea-journal/README.md

```diff
@@ -0,0 +1,54 @@
+# React Fundamentals I
```

Comment:
any reason for new file and not just edit `lesson-2/README.md`?

Reply:
yeah, please rewrite lesson-2/README.md rather, so every lesson keeps the same structure :-))) thanks!